test/regression/api/canary/admin/utils.js

```diff
@@ -86,7 +86,6 @@ const expectedProperties = {
     member: _(schema.members)
         .keys()
         .concat('avatar_image')
-        .concat('comped')
```

Comment:
As this is been touched can you please follow up with a small refactor into "allowlist" syntax as done [here](https://github.com/TryGhost/Ghost/commit/06dd9bac5957d4cd3415a3086c275b2a7501fe3e)? :raised_hands:

Reply:
Done!